linuxkpi/gplv2/include/uapi/linux/eventpoll.h

```diff
@@ -0,0 +1,20 @@
+#ifndef _UAPI_LINUX_EVENTPOLL_H
+#define	_UAPI_LINUX_EVENTPOLL_H
+
+#include <linux/types.h>
+
+/* Epoll event masks */
+#define EPOLLIN		(__force __poll_t)0x00000001
```

Comment:
Don't use Linux values here as these constants are exposed to userworld which expects FreeBSD poll() compatibility.
Just include sys/poll.h and aliase them to FreeBSD POLL* definitions.
Also this file should be linux/eventpoll.h rather than uapi/linux/eventpoll.h